website/docs/cloud-docs/stacks/deploy/plans.mdx

```diff
@@ -50,7 +53,7 @@ A Stack deployment can have multiple plans. In a deployment's page, underneath *
 
 You can see a plan’s full details by clicking the plan’s name or an abbreviated list of the plan’s changes by clicking **Quick View**. You can also click **View all plans** to display a list of all the plans for this deployment. You can filter plans by [health status](#check-deployment-health) in the list of all plans. 
 
-Each plan includes a timeline detailing when the plan started, when it received its configuration version, and when it was approved.  
+Each plan includes the Stack’s inputs and outputs, a timeline detailing when the plan started, when the plan received its configuration version, and when it was approved.  
```

Comment:
A plan belongs to a single deployment within a stack, so the outputs of plan might not necessary represent the entire stack output. It represents the output of a single deployment.  Same with inputs, the inputs consumed by a deployment plan may be a subset of the entire stack input.

Reply:
Thank you for explaining that context, so this would be more accurate, correct? 
```suggestion
Each plan includes that plan's inputs and outputs, a timeline detailing when the plan started, when the plan received its configuration version, and when it was approved.  
```

Also I'd love to know if you feel like I'm missing anything else here in terms of conveying where else in the UI inputs and outputs show up?

Reply:
this update looks good to me